workers/reporter/src/main/kotlin/reporter/SourceCodeBundleReporter.kt

```diff
@@ -94,13 +114,23 @@ class SourceCodeBundleReporter(
             }
         }
 
+        val filteredPackages = allPackages.takeIf { includedLicenseCategories.isEmpty() } ?: allPackages.filter { pkg ->
+            val pkgLicenseCategories = pkg.getLicenseCategories(
+                licenseClassifications.categorizations,
+                ortResult.createLicenseInfoResolver(),
```

Comment:
Not sure what exactly `ortResult.createLicenseInfoResolver()` does, but I assume such a resolver should be created only once and not for each element to be filtered.